src/constants/errorWrapperOptions.ts

```diff
@@ -0,0 +1,22 @@
+import { RouteHandlerErrorProperties } from '../types';
+
+export const DEFAULT_CUSTOM_LOG_FUNCTION = ({
+  method,
+  originalUrl,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
+  statusCode,
+  message,
+}: RouteHandlerErrorProperties) => {
+  console.error(`REQUEST ERROR: [${method}] ${originalUrl}: ${message}`);
+};
+export const DEFAULT_CUSTOM_JSON_RESPONSE = ({
+  method,
+  originalUrl,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
+  statusCode,
```

Comment:
You don't need to destructure statusCode 

```suggestion
```

Reply:
Same as above.